app.js

```diff
@@ -0,0 +1,87 @@
+var mode = prompt("Input mode: (b)asic, (a)dvanced, (bmi) calculator, (t)rip planner");
+switch (mode) {
+  case 'b':
+    console.log("Basic");
+    var action = prompt("Input a basic sum with two integers, seperated by space, and using +, -, *, or /");
+    var calc = action.split(" ");
```

Comment:
What you're doing here is really interesting and well-implemented, but not that clear from a quick glance. I would probably use some comments to explain what this code is doing.